sdk/python/foundation-models/system/inference/text-to-image/safe-text-to-image-online-deployment.ipynb

```diff
@@ -0,0 +1,601 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to create an Azure AI Content Safety enabled text-to-image online endpoint (Preview)\n",
+    "### This notebook will walk you through the steps to create an __Azure AI Content Safety__ enabled __text-to-image__ online endpoint.\n",
+    "### This notebook is under preview\n",
+    "### The steps are:\n",
```

Comment:
Comments in here are similar to what were provided for this PR https://github.com/Azure/azureml-examples/pull/2629

Reply:
Addressed it.